open_lab_notebook/Dez_update

```diff
@@ -0,0 +1,24 @@
+# Dezember
+
+## Flashback on last months
+- only got DICOM need to transform data into BIDS
+- manage to open Matlab files to get info about reccordings etc.
+
+## Week 1
+- wasn't available because of conference
+
+## Week 2
+- Corona hit me
+- searching for code to open Matlab-files-> used pymatreader -> did work, but no idea if that is what i needed
```

Comment:
I usually use [`scipy`'s `io.loadmat`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.io.loadmat.html) function like so

```
import scipy.io
mat = scipy.io.loadmat('file.mat') 
```

However, if your approach works well, all good!

Reply:
I got a nice data frame with this :)

`from pymatreader import read_mat
import pandas as pd

data = read_mat('Trials_S01_1.mat') 
data.keys()
mat= pd.DataFrame(data['Trials']) 
mat`